Missing python flask dependency package markupsafe

In the present release [`v0.0.4`](https://github.com/theohbrothers/spotify-connect-web/releases/tag/v0.0.4), `spotify-connect-web` would not start without the package `marksafe`.

```
spotify-connect-web    | Traceback (most recent call last):
spotify-connect-web    |   File "main.py", line 10, in <module>
spotify-connect-web    |     from flask_bootstrap import Bootstrap
spotify-connect-web    |   File "/usr/local/lib/python2.7/dist-packages/flask_bootstrap/__init__.py", line 20, in <module>
spotify-connect-web    |     from .forms import render_form
spotify-connect-web    |   File "/usr/local/lib/python2.7/dist-packages/flask_bootstrap/forms.py", line 4, in <module>
spotify-connect-web    |     from markupsafe import Markup
spotify-connect-web    | ImportError: No module named markupsafe
```

Based on the release date of [`spotify-connect-web` `0.0.4-alpha`](https://github.com/Fornoth/spotify-connect-web/releases/tag/0.0.4-alpha), the package [`markupsafe` `1.0`](https://github.com/pallets/markupsafe/releases/tag/1.0) is likely compatible based on the latest release (https://github.com/pallets/markupsafe/tags) prior to the release of `spotify-connect-web` `0.0.4-alpha`.

So I seemed to have circumvented the error with:

```
$ pip install markupsafe==1.0
```

The following is the output I get from installing just `Jinja2==2.7` with its dependencies on top of the `v0.0.4` image via `pip install --upgrade jinja2==2.7`:

```sh
docker-compose up --force-recreate
Recreating spotify-connect-web ... done
Attaching to spotify-connect-web
spotify-connect-web    | Downloading/unpacking jinja2==2.7
spotify-connect-web    |   Running setup.py (path:/tmp/pip-build-y9hCYS/jinja2/setup.py) egg_info for package jinja2
spotify-connect-web    |     
spotify-connect-web    |     warning: no files found matching '*' under directory 'custom_fixers'
spotify-connect-web    |     warning: no previously-included files matching '*' found under directory 'docs/_build'
spotify-connect-web    |     warning: no previously-included files matching '*.pyc' found under directory 'jinja2'
spotify-connect-web    |     warning: no previously-included files matching '*.pyc' found under directory 'docs'
spotify-connect-web    |     warning: no previously-included files matching '*.pyo' found under directory 'jinja2'
spotify-connect-web    |     warning: no previously-included files matching '*.pyo' found under directory 'docs'
spotify-connect-web    | Downloading/unpacking markupsafe (from jinja2==2.7)
spotify-connect-web    |   Downloading MarkupSafe-2.0.1.tar.gz
spotify-connect-web    |   Running setup.py (path:/tmp/pip-build-y9hCYS/markupsafe/setup.py) egg_info for package markupsafe
spotify-connect-web    |     
spotify-connect-web    |     no previously-included directories found matching 'docs/_build'
spotify-connect-web    |     warning: no previously-included files matching '*.pyc' found anywhere in distribution
spotify-connect-web    | Installing collected packages: jinja2, markupsafe
spotify-connect-web    |   Found existing installation: Jinja2 2.4.1
spotify-connect-web    |     Uninstalling Jinja2:
spotify-connect-web    |       Successfully uninstalled Jinja2
spotify-connect-web    |   Running setup.py install for jinja2
spotify-connect-web    |     
spotify-connect-web    |     warning: no files found matching '*' under directory 'custom_fixers'
spotify-connect-web    |     warning: no previously-included files matching '*' found under directory 'docs/_build'
spotify-connect-web    |     warning: no previously-included files matching '*.pyc' found under directory 'jinja2'
spotify-connect-web    |     warning: no previously-included files matching '*.pyc' found under directory 'docs'
spotify-connect-web    |     warning: no previously-included files matching '*.pyo' found under directory 'jinja2'
spotify-connect-web    |     warning: no previously-included files matching '*.pyo' found under directory 'docs'
spotify-connect-web    |   Running setup.py install for markupsafe
spotify-connect-web    |     building 'markupsafe._speedups' extension
spotify-connect-web    |     arm-linux-gnueabi-gcc -pthread -DNDEBUG -g -fwrapv -O2 -Wall -Wstrict-prototypes -fno-strict-aliasing -D_FORTIFY_SOURCE=2 -g -fstack-protector-strong -Wformat -Werror=format-security -fPIC -I/usr/include/python2.7 -c src/markupsafe/_speedups.c -o build/temp.linux-armv7l-2.7/src/markupsafe/_speedups.o
spotify-connect-web    |     src/markupsafe/_speedups.c: In function 'escape_unicode_kind1':
spotify-connect-web    |     src/markupsafe/_speedups.c:94:2: error: unknown type name 'Py_UCS1'
spotify-connect-web    |       Py_UCS1 *inp = PyUnicode_1BYTE_DATA(in);
spotify-connect-web    |       ^
spotify-connect-web    |     src/markupsafe/_speedups.c:94:2: warning: implicit declaration of function 'PyUnicode_1BYTE_DATA' [-Wimplicit-function-declaration]
spotify-connect-web    |     src/markupsafe/_speedups.c:94:17: warning: initialization makes pointer from integer without a cast
spotify-connect-web    |       Py_UCS1 *inp = PyUnicode_1BYTE_DATA(in);
spotify-connect-web    |                      ^
spotify-connect-web    |     src/markupsafe/_speedups.c:95:2: error: unknown type name 'Py_UCS1'
spotify-connect-web    |       Py_UCS1 *inp_end = inp + PyUnicode_GET_LENGTH(in);
spotify-connect-web    |       ^
spotify-connect-web    |     src/markupsafe/_speedups.c:95:2: warning: implicit declaration of function 'PyUnicode_GET_LENGTH' [-Wimplicit-function-declaration]
spotify-connect-web    |     src/markupsafe/_speedups.c:96:2: error: unknown type name 'Py_UCS1'
spotify-connect-web    |       Py_UCS1 *outp;
spotify-connect-web    |       ^
spotify-connect-web    |     src/markupsafe/_speedups.c:106:2: warning: implicit declaration of function 'PyUnicode_New' [-Wimplicit-function-declaration]
spotify-connect-web    |       out = PyUnicode_New(PyUnicode_GET_LENGTH(in) + delta,
spotify-connect-web    |       ^
spotify-connect-web    |     src/markupsafe/_speedups.c:107:7: warning: implicit declaration of function 'PyUnicode_IS_ASCII' [-Wimplicit-function-declaration]
spotify-connect-web    |            PyUnicode_IS_ASCII(in) ? 127 : 255);
spotify-connect-web    |            ^
spotify-connect-web    |     src/markupsafe/_speedups.c:106:6: warning: assignment makes pointer from integer without a cast
spotify-connect-web    |       out = PyUnicode_New(PyUnicode_GET_LENGTH(in) + delta,
spotify-connect-web    |           ^
spotify-connect-web    |     src/markupsafe/_speedups.c:111:6: warning: assignment makes pointer from integer without a cast
spotify-connect-web    |       inp = PyUnicode_1BYTE_DATA(in);
spotify-connect-web    |           ^
spotify-connect-web    |     src/markupsafe/_speedups.c:112:7: warning: assignment makes pointer from integer without a cast
spotify-connect-web    |       outp = PyUnicode_1BYTE_DATA(out);
spotify-connect-web    |            ^
spotify-connect-web    |     src/markupsafe/_speedups.c: In function 'escape_unicode_kind2':
spotify-connect-web    |     src/markupsafe/_speedups.c:120:2: error: unknown type name 'Py_UCS2'
spotify-connect-web    |       Py_UCS2 *inp = PyUnicode_2BYTE_DATA(in);
spotify-connect-web    |       ^
spotify-connect-web    |     src/markupsafe/_speedups.c:120:2: warning: implicit declaration of function 'PyUnicode_2BYTE_DATA' [-Wimplicit-function-declaration]
spotify-connect-web    |     src/markupsafe/_speedups.c:120:17: warning: initialization makes pointer from integer without a cast
spotify-connect-web    |       Py_UCS2 *inp = PyUnicode_2BYTE_DATA(in);
spotify-connect-web    |                      ^
spotify-connect-web    |     src/markupsafe/_speedups.c:121:2: error: unknown type name 'Py_UCS2'
spotify-connect-web    |       Py_UCS2 *inp_end = inp + PyUnicode_GET_LENGTH(in);
spotify-connect-web    |       ^
spotify-connect-web    |     src/markupsafe/_speedups.c:122:2: error: unknown type name 'Py_UCS2'
spotify-connect-web    |       Py_UCS2 *outp;
spotify-connect-web    |       ^
spotify-connect-web    |     src/markupsafe/_speedups.c:132:6: warning: assignment makes pointer from integer without a cast
spotify-connect-web    |       out = PyUnicode_New(PyUnicode_GET_LENGTH(in) + delta, 65535);
spotify-connect-web    |           ^
spotify-connect-web    |     src/markupsafe/_speedups.c:136:6: warning: assignment makes pointer from integer without a cast
spotify-connect-web    |       inp = PyUnicode_2BYTE_DATA(in);
spotify-connect-web    |           ^
spotify-connect-web    |     src/markupsafe/_speedups.c:137:7: warning: assignment makes pointer from integer without a cast
spotify-connect-web    |       outp = PyUnicode_2BYTE_DATA(out);
spotify-connect-web    |            ^
spotify-connect-web    |     src/markupsafe/_speedups.c: In function 'escape_unicode_kind4':
spotify-connect-web    |     src/markupsafe/_speedups.c:146:2: warning: implicit declaration of function 'PyUnicode_4BYTE_DATA' [-Wimplicit-function-declaration]
spotify-connect-web    |       Py_UCS4 *inp = PyUnicode_4BYTE_DATA(in);
spotify-connect-web    |       ^
spotify-connect-web    |     src/markupsafe/_speedups.c:146:17: warning: initialization makes pointer from integer without a cast
spotify-connect-web    |       Py_UCS4 *inp = PyUnicode_4BYTE_DATA(in);
spotify-connect-web    |                      ^
spotify-connect-web    |     src/markupsafe/_speedups.c:158:6: warning: assignment makes pointer from integer without a cast
spotify-connect-web    |       out = PyUnicode_New(PyUnicode_GET_LENGTH(in) + delta, 1114111);
spotify-connect-web    |           ^
spotify-connect-web    |     src/markupsafe/_speedups.c:162:6: warning: assignment makes pointer from integer without a cast
spotify-connect-web    |       inp = PyUnicode_4BYTE_DATA(in);
spotify-connect-web    |           ^
spotify-connect-web    |     src/markupsafe/_speedups.c:163:7: warning: assignment makes pointer from integer without a cast
spotify-connect-web    |       outp = PyUnicode_4BYTE_DATA(out);
spotify-connect-web    |            ^
spotify-connect-web    |     src/markupsafe/_speedups.c: In function 'escape_unicode':
spotify-connect-web    |     src/markupsafe/_speedups.c:171:2: warning: implicit declaration of function 'PyUnicode_READY' [-Wimplicit-function-declaration]
spotify-connect-web    |       if (PyUnicode_READY(in))
spotify-connect-web    |       ^
spotify-connect-web    |     src/markupsafe/_speedups.c:174:2: warning: implicit declaration of function 'PyUnicode_KIND' [-Wimplicit-function-declaration]
spotify-connect-web    |       switch (PyUnicode_KIND(in)) {
spotify-connect-web    |       ^
spotify-connect-web    |     src/markupsafe/_speedups.c:175:7: error: 'PyUnicode_1BYTE_KIND' undeclared (first use in this function)
spotify-connect-web    |       case PyUnicode_1BYTE_KIND:
spotify-connect-web    |            ^
spotify-connect-web    |     src/markupsafe/_speedups.c:175:7: note: each undeclared identifier is reported only once for each function it appears in
spotify-connect-web    |     src/markupsafe/_speedups.c:177:7: error: 'PyUnicode_2BYTE_KIND' undeclared (first use in this function)
spotify-connect-web    |       case PyUnicode_2BYTE_KIND:
spotify-connect-web    |            ^
spotify-connect-web    |     src/markupsafe/_speedups.c:179:7: error: 'PyUnicode_4BYTE_KIND' undeclared (first use in this function)
spotify-connect-web    |       case PyUnicode_4BYTE_KIND:
spotify-connect-web    |            ^
spotify-connect-web    |     src/markupsafe/_speedups.c: In function 'escape':
spotify-connect-web    |     src/markupsafe/_speedups.c:193:3: warning: implicit declaration of function 'PyUnicode_InternFromString' [-Wimplicit-function-declaration]
spotify-connect-web    |        id_html = PyUnicode_InternFromString("__html__");
spotify-connect-web    |        ^
spotify-connect-web    |     src/markupsafe/_speedups.c:193:11: warning: assignment makes pointer from integer without a cast
spotify-connect-web    |        id_html = PyUnicode_InternFromString("__html__");
spotify-connect-web    |                ^
spotify-connect-web    |     src/markupsafe/_speedups.c: At top level:
spotify-connect-web    |     src/markupsafe/_speedups.c:320:15: error: variable 'module_definition' has initializer but incomplete type
spotify-connect-web    |      static struct PyModuleDef module_definition = {
spotify-connect-web    |                    ^
spotify-connect-web    |     src/markupsafe/_speedups.c:321:2: error: 'PyModuleDef_HEAD_INIT' undeclared here (not in a function)
spotify-connect-web    |       PyModuleDef_HEAD_INIT,
spotify-connect-web    |       ^
spotify-connect-web    |     src/markupsafe/_speedups.c:321:2: warning: excess elements in struct initializer
spotify-connect-web    |     src/markupsafe/_speedups.c:321:2: warning: (near initialization for 'module_definition')
spotify-connect-web    |     src/markupsafe/_speedups.c:322:2: warning: excess elements in struct initializer
spotify-connect-web    |       "markupsafe._speedups",
spotify-connect-web    |       ^
spotify-connect-web    |     src/markupsafe/_speedups.c:322:2: warning: (near initialization for 'module_definition')
spotify-connect-web    |     src/markupsafe/_speedups.c:323:2: warning: excess elements in struct initializer
spotify-connect-web    |       NULL,
spotify-connect-web    |       ^
spotify-connect-web    |     src/markupsafe/_speedups.c:323:2: warning: (near initialization for 'module_definition')
spotify-connect-web    |     src/markupsafe/_speedups.c:324:2: warning: excess elements in struct initializer
spotify-connect-web    |       -1,
spotify-connect-web    |       ^
spotify-connect-web    |     src/markupsafe/_speedups.c:324:2: warning: (near initialization for 'module_definition')
spotify-connect-web    |     src/markupsafe/_speedups.c:325:2: warning: excess elements in struct initializer
spotify-connect-web    |       module_methods,
spotify-connect-web    |       ^
spotify-connect-web    |     src/markupsafe/_speedups.c:325:2: warning: (near initialization for 'module_definition')
spotify-connect-web    |     src/markupsafe/_speedups.c:326:2: warning: excess elements in struct initializer
spotify-connect-web    |       NULL,
spotify-connect-web    |       ^
spotify-connect-web    |     src/markupsafe/_speedups.c:326:2: warning: (near initialization for 'module_definition')
spotify-connect-web    |     src/markupsafe/_speedups.c:327:2: warning: excess elements in struct initializer
spotify-connect-web    |       NULL,
spotify-connect-web    |       ^
spotify-connect-web    |     src/markupsafe/_speedups.c:327:2: warning: (near initialization for 'module_definition')
spotify-connect-web    |     src/markupsafe/_speedups.c:328:2: warning: excess elements in struct initializer
spotify-connect-web    |       NULL,
spotify-connect-web    |       ^
spotify-connect-web    |     src/markupsafe/_speedups.c:328:2: warning: (near initialization for 'module_definition')
spotify-connect-web    |     src/markupsafe/_speedups.c:330:1: warning: excess elements in struct initializer
spotify-connect-web    |      };
spotify-connect-web    |      ^
spotify-connect-web    |     src/markupsafe/_speedups.c:330:1: warning: (near initialization for 'module_definition')
spotify-connect-web    |     src/markupsafe/_speedups.c: In function 'PyInit__speedups':
spotify-connect-web    |     src/markupsafe/_speedups.c:336:3: warning: 'return' with a value, in function returning void
spotify-connect-web    |        return NULL;
spotify-connect-web    |        ^
spotify-connect-web    |     src/markupsafe/_speedups.c:338:2: warning: implicit declaration of function 'PyModule_Create' [-Wimplicit-function-declaration]
spotify-connect-web    |       return PyModule_Create(&module_definition);
spotify-connect-web    |       ^
spotify-connect-web    |     src/markupsafe/_speedups.c:338:2: warning: 'return' with a value, in function returning void
spotify-connect-web    |     ==========================================================================
spotify-connect-web    |     WARNING: The C extension could not be compiled, speedups are not enabled.
spotify-connect-web    |     Failure information, if any, is above.
spotify-connect-web    |     Retrying the build without the C extension now.
spotify-connect-web    |     ==========================================================================
spotify-connect-web    |
spotify-connect-web    |     no previously-included directories found matching 'docs/_build'
spotify-connect-web    |     warning: no previously-included files matching '*.pyc' found anywhere in distribution
spotify-connect-web    |     ==========================================================================
spotify-connect-web    |     WARNING: The C extension could not be compiled, speedups are not enabled.
spotify-connect-web    |     Plain-Python build succeeded.
spotify-connect-web    |     ==========================================================================
spotify-connect-web    | Successfully installed jinja2 markupsafe
spotify-connect-web    | Cleaning up...
spotify-connect-web    | Flask==0.11.1
spotify-connect-web    | Flask-Bootstrap==3.3.7.1
spotify-connect-web    | Flask-Cors==2.1.2
spotify-connect-web    | Jinja2==2.7
spotify-connect-web    | MarkupSafe==0.0.0
spotify-connect-web    | Werkzeug==0.16.1
spotify-connect-web    | argparse==1.2.1
spotify-connect-web    | cffi==0.9.2
spotify-connect-web    | chardet==2.3.0
spotify-connect-web    | click==2.0
spotify-connect-web    | colorama==0.3.2
spotify-connect-web    | cryptography==0.6.1
spotify-connect-web    | dominate==2.6.0
spotify-connect-web    | gevent==1.0.1
spotify-connect-web    | greenlet==0.4.2
spotify-connect-web    | html5lib==0.999
spotify-connect-web    | itsdangerous==0.21
spotify-connect-web    | ndg-httpsclient==0.3.2
spotify-connect-web    | ply==3.4
spotify-connect-web    | pyOpenSSL==0.14
spotify-connect-web    | pyalsaaudio==0.8.4
spotify-connect-web    | pyasn1==0.1.7
spotify-connect-web    | pycparser==2.10
spotify-connect-web    | pylast==1.6.0
spotify-connect-web    | requests==2.4.3
spotify-connect-web    | six==1.8.0
spotify-connect-web    | urllib3==1.9.1
spotify-connect-web    | visitor==0.1.3
spotify-connect-web    | wheel==0.24.0
spotify-connect-web    | wsgiref==0.1.2
spotify-connect-web    | Traceback (most recent call last):
spotify-connect-web    |   File "main.py", line 9, in <module>
spotify-connect-web    |     from flask import Flask, request, abort, jsonify, render_template, redirect, flash, url_for
spotify-connect-web    |   File "/usr/local/lib/python2.7/dist-packages/flask/__init__.py", line 19, in <module>
spotify-connect-web    |     from jinja2 import Markup, escape
spotify-connect-web    |   File "/usr/local/lib/python2.7/dist-packages/jinja2/__init__.py", line 33, in <module>
spotify-connect-web    |     from jinja2.environment import Environment, Template
spotify-connect-web    |   File "/usr/local/lib/python2.7/dist-packages/jinja2/environment.py", line 13, in <module>
spotify-connect-web    |     from jinja2 import nodes
spotify-connect-web    |   File "/usr/local/lib/python2.7/dist-packages/jinja2/nodes.py", line 18, in <module>
spotify-connect-web    |     from jinja2.utils import Markup
spotify-connect-web    |   File "/usr/local/lib/python2.7/dist-packages/jinja2/utils.py", line 520, in <module>
spotify-connect-web    |     from markupsafe import Markup, escape, soft_unicode
spotify-connect-web    | ImportError: No module named markupsafe
spotify-connect-web exited with code 1
```

Despite the installer claiming `markupsafe` is installed, `pip freeze` outputs the installed version as `MarkupSafe==0.0.0`.

@leojonathanoh

--- a/requirements.txt
+++ b/requirements.txt
@@ -10,3 +10,4 @@ werkzeug<1.0
 Jinja2>=2.7,<3.0
 itsdangerous>=0.21,<0.22
 click>=2.0,<2.1
+markupsafe==1.0

